apps/ledger-live-desktop/src/renderer/families/elrond/components/Modals/Undelegate/Body.jsx

```diff
@@ -88,20 +88,17 @@ const Body = (props: Props) => {
     const bridge: AccountBridge<Transaction> = getAccountBridge(accountProp, undefined);
     const transaction: Transaction = bridge.createTransaction(accountProp);
 
-    bridge.updateTransaction(transaction, {
-      mode: "unDelegate",
-    });
-
     return {
       account: accountProp,
-      transaction: contract
-        ? bridge.updateTransaction(transaction, {
-            recipient: contract,
-          })
-        : transaction,
+      transaction: bridge.updateTransaction(transaction, {
+        recipient: contract || "",
+        mode: "unDelegate",
+      }),
     };
   });
 
+  console.log("UNDELEGATING", transaction);
+
```

Comment:
Should be removed :slightly_smiling_face:

Reply:
Removed in our latest version. 👍